### PR TITLE
Add probot stale.yml

### PR DESCRIPTION
**Pull request by jesseproudman**

### CODE REVIEW

File: .github/.stale.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (devinburnette) [inline .github/.stale.yml:1]: i think the file name should be `stale.yml` without the dot

Comment (devinburnette) [reply .github/.stale.yml:1]: not sure if it matters

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```
Comment (devinburnette) [inline .github/stale.yml:6]: do we need something here or is it a valid key empty like this? maybe we can just remove?

Comment (jesseproudman) [reply .github/stale.yml]: Adjusted

**Review by devinburnette — APPROVED** [review]
domain lgtm && platform lgtm

**Pull request merged**